cvat-ui/src/utils/mousetrap-react.tsx

```diff
@@ -52,6 +52,26 @@ export default function GlobalHotKeys(props: Props): JSX.Element {
     return children || <></>;
 }
 
+Mousetrap.prototype.stopCallback = function (e: KeyboardEvent, element: Element, combo: string): boolean {
+    // if the element has the class "mousetrap" then no need to stop
+    if ((` ${(element as HTMLElement).className} `).indexOf(' mousetrap ') > -1) {
+        return false;
+    }
```

Comment:
What elements have "mousetrap" class?
Do we really need those spaces?
```suggestion
    // if the element has the class "mousetrap" then no need to stop
    if ((`${(element as HTMLElement).className}`).indexOf(' mousetrap ') > -1) {
        return false;
    }
```

Reply:
@klakhov 

This is a part of default implementation (can be found here: https://craig.is/killing/mice)
According to it, elements having "mousetrap" class trigger shortcuts handlers (but we do not actually use such elements).
Spaces are necessary to fix case when "mousetrap" class is in the beginning, or at the end.

Reply:
Removed, since not used